### ISSUE TITLE
vlc crushes when vlsub run on any custom skins

### ISSUE THREAD

**Issue by sfzaw** [exebetche/vlsub#181]
just as title says. OS :win7 starter

**Comment by ThatLurker** [comment]
What skin are you using?

**Comment by sfzaw** [comment]
minimalX currently. tried arc, e dark etc. when i clicked vlsub vlc crushed with all types of skin. then when i switched to natives it works perfectly.

**Comment by Abhinav1217** [comment]
For me VLC is crashing with vlsub even with default skin.

**Comment by ThatLurker** [comment]
@Abhinav1217 are you using the latest version of vlc and vlsub

**Comment by Abhinav1217** [comment]
Yes. Fresh installed Fedora 26 three days ago I run system update daily,  VLC installed using DNF, current version of "3.0.0-git Vetinari", vlsub is 0.10.2.  Downloaded from github yesterday. I was having issues in F25 also, for past few months. 

I Think this can also be related to slow downloading issue, becase it gets stuck at 100% and then it is the system message for not responding program vlsub. If i press any button, like wait or force quit, VLC crashes, But I could just choose to ignore it and continue watching movie as long as I dont need any controls.

After reading issues here, I checked my folder, VLSub did download the <subtitle>.gz file which contained the subtitle, but extracted srt file was 0 bytes. Manually extracted subtitle was fine though.

**Comment by ThatLurker** [comment]
The git version of vlc is causing some issues for some reason

**Comment by Abhinav1217** [comment]
Ok, that makes some sense. Also turns out that 3.0.0 is still preview release according to website. Looks like I am stuck with it for some time.  Thanks for your time.

**Comment by wolfyllow** [comment]
Same problem with:
- VLC 2.2.6 
- VLSub 0.10.2
- Windows 10

**Comment by Abhinav1217** [comment]
Turns out It is not related to the git release of VLC.   It is related to Wayland which is now default in Fedora.

**Comment by TripleEYE** [comment]
Any news on this? it's been quite a while and this still seems to be an active bug (at least on my PC with MinimalX skin and VLSUB 10 and VLSUB 11)...
would love to know this has been resolved and there's some simple solution either from VLSUB side or by somehow re-configuring something in the skin manually (as a user)
